extension-style/src/main/java/com/mapbox/maps/extension/style/StyleExtensionImpl.kt

```diff
@@ -145,6 +147,10 @@ class StyleExtensionImpl private constructor(
     /**
      * Extension function for [Projection] to overload Unary operations.
      *
+     * Using [ProjectionName.GLOBE] requires OpenGL [GLES20.GL_MAX_VERTEX_TEXTURE_IMAGE_UNITS] be more than zero.
+     * If [GLES20.GL_MAX_VERTEX_TEXTURE_IMAGE_UNITS] is zero -
+     * [ProjectionName.GLOBE] will fallback to [ProjectionName.MERCATOR] with the log warning.
```

Comment:
shall we mention these typically means older/legacy devices?

Reply:
done!